packages/angular_devkit/schematics/tools/index.ts

```diff
@@ -13,3 +13,6 @@ export { FallbackEngineHost } from './fallback-engine-host';
 export { FileSystemEngineHost } from './file-system-engine-host';
 export { NodeModulesEngineHost } from './node-module-engine-host';
 export { NodeModulesTestEngineHost } from './node-modules-test-engine-host';
+
+export { AjvSchemaRegistry } from './ajv-option-transform';
```

Comment:
I think we're going to have to move this to core if we want to share code with Build Facade or other things.

Reply:
Yes.

Reply:
Please do this tomorrow :)